tests/DfE.FindInformationAcademiesTrusts.UnitTests/Services/AcademyServiceTests.cs

```diff
@@ -60,4 +61,37 @@ public async Task GetAcademiesInTrustOfstedAsync_should_return_mapped_result_fro
         result.Should().BeOfType<AcademyOfstedServiceModel[]>();
         result.Should().BeEquivalentTo(academies);
     }
+
+    [Fact]
+    public async Task GetAcademiesInTrustPupilNumbersAsync_should_return_mapped_result_from_repository()
+    {
+        const string uid = "1234";
+        AcademyPupilNumbers[] academies =
+        [
+            BuildDummyAcademyPupilNumbers("9876"),
+            BuildDummyAcademyPupilNumbers("9876")
```

Comment:
These two academies are identical so this test doesn't really test that things are mapped correctly, just that you get this particular result back. I'd make the two academies have every field different